modin/engines/ray/utils.py

```diff
@@ -165,6 +165,7 @@ def initialize_ray(
                 "logging_level": 100,
                 "_memory": object_store_memory,
                 "_lru_evict": True,
+                "log_to_driver": False,
```

Comment:
how come a code change is stuffed into docs PR?.. if you find a bug during reading please mark it with either a github issue or a `#TODO` / `#FIXME` comment, do not mix multiple things in a single PR

Reply:
This is debug code, now it's removed. It was added to suppress excessive logs from ray workers. There is no need to create new issue since this is known issue in the ray.